doc/book/navigation.md

```diff
@@ -232,7 +232,7 @@ written:
             ->breadcrumbs()
             ->setMinDepth(0)
             ->setPartial('partial/breadcrumb') ?>
-    <?= $this->content ?>
+    <?= $this->content; ?>
```

Comment:
Look to the line above and you will see no semicolon. The whole document is inconsistent.
Please undo.